### PR TITLE
Sensible message : Merging features X,Y,Z to main branch

### PR DESCRIPTION
**Pull request by Saujordan20**

### CODE REVIEW

**Comment by MU5T4F4** [comment]
Why was this made

**Review by chakri007-kc — APPROVED** [review]

**Review by RohiniYipli — COMMENTED** [review]
Reviewed

**Comment by MU5T4F4** [comment]
> Reviewed

When

File: Mustafa.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-If you are reading this file send help
```
Comment (Saujordan20) [inline Mustafa.txt:1]: Can you optimize this code.
It has very high Time complexity.

**Review by RohiniYipli — APPROVED** [review]

**Review by MU5T4F4 — APPROVED** [review]
Thanks for the comment

**Pull request merged**